### PR TITLE
Bump SDK packages and minor typing fixes

### PR DESCRIPTION
**Pull request by austinkelleher**

### CODE REVIEW

**Review by aiwilliams — APPROVED** [review]

File: src/utils/project.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import {
+  Entity,
+  IntegrationError,
+  JobState,
+} from '@jupiterone/integration-sdk-core';
+import { PROJECT_ENTITY_TYPE } from '../steps/resource-manager';
+
+export async function getProjectEntity(jobState: JobState): Promise<Entity> {
```
Comment (ndowmon) [inline src/utils/project.ts:8]: Sweet!

**Review by ndowmon — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,34 @@ and this project adheres to
 
 ## [Unreleased]
 
+## 0.15.0 - 2020-02-23
```
Comment (ndowmon) [inline CHANGELOG.md:11]: Oops. Didn't realize the date was wrong here. Fix it later though.

Comment (ndowmon) [reply CHANGELOG.md:11]: I remember February 2020... :longing:

**Review by ndowmon — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (austinkelleher) [reply CHANGELOG.md:11]: RIP. Will fix soon. Thanks.

**Pull request merged**